HomeWork_7/Command.java

```diff
@@ -0,0 +1,14 @@
+import java.util.ArrayList;
+
+/**
+ * Base class Command has only one method performCommand
+ *
+ * @author Nikita Zenchik
+ */
+public abstract class Command {
+
+    /**
+     * Method performCommand will be overridden
+     */
+    public abstract void checkCommand(String line, ArrayList<Product> list);
```

Comment:
Comment is for performCommand but method name is checkCommand.
One more thing - avoid repeating obvious thing (Command) in the method names